internal/cloud/realm/dependencies.go

```diff
@@ -18,10 +19,43 @@ const (
 	dependenciesPathPattern        = appPathPattern + "/dependencies"
 	dependenciesArchivePathPattern = dependenciesPathPattern + "/archive"
 	dependenciesDiffPathPattern    = dependenciesPathPattern + "/diff"
+	dependenciesStatusPathPattern  = dependenciesPathPattern + "/status"
 
 	paramFile = "file"
 )
 
+// DependencyInstallation is used to get information from a dependencies status request
+type DependencyInstallation struct {
+	Status        DependenciesStatus `json:"status"`
+	StatusMessage DependenciesStatus `json:"status_message"`
+}
+
+// DependenciesStatus represents possible dependency statuses
+type DependenciesStatus string
+
+// set of known dependencies statuses
+const (
+	DependenciesStatusCreated    DependenciesStatus = "created"
+	DependenciesStatusSuccessful DependenciesStatus = "successful"
+	DependenciesStatusFailed     DependenciesStatus = "failed"
+)
+
+func (c *client) GetDependenciesStatus(groupID, appID string) (DependenciesStatus, error) {
+	res, err := c.do(
+		http.MethodGet,
+		fmt.Sprintf(dependenciesStatusPathPattern, groupID, appID),
+		api.RequestOptions{},
+	)
+	if err != nil {
+		return "", err
+	}
```

Comment:
two other things we'll want to do here (as you'll see in other client method implementations):

1. check the `res.StatusCode` is what we expect it to be
  - return `api.ErrUnexpectedStatus{"get dependencies status", res.StatusCode}` if it doesnt match
2. after that, you can `defer res.Body.Close()`
  - b/c we're then consuming the response body, we want to make sure we close it